src/components/ui/HudContainer.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { createPortal } from "react-dom";
+
+/**
+ * Heads up display container which portals all Hud Components out to the body and applies safe area styling - should be used for all game Hud components
+ */
+export const HudContainer: React.FC = ({ children }) => {
```

Comment:
In order to make sure mobile safe areas are considered and handled correctly all hud's will now be wrapped in a `HudContainer`